sql/src/ast.rs

```diff
@@ -69,6 +69,19 @@ pub struct CreateTable {
     pub constraints: Vec<TableConstraint>,
     /// Table options in `WITH`.
     pub options: Vec<SqlOption>,
+    pub partition: Option<Partition>,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Partition {
+    Hash(HashPartition),
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct HashPartition {
+    pub linear: bool,
```

Comment:
Add some comments  to explain the use of `linear`.

Reply:
Done.